cmd/check_container.go

```diff
@@ -242,3 +243,16 @@ func buildConnectURL(projectID string) string {
 
 	return connectURL
 }
+
+func buildScanResultsURL(projectID string, imageID string) string {
+	connectURL := fmt.Sprintf("https://connect.redhat.com/projects/%s/images/%s/scan-results", projectID, imageID)
```

Comment:
You could factor out a tunction that builds the "connect" URL... And just tack on the extra bit at the end for each.

Reply:
To be addressed in another PR.

/lgtm